rel/config.toml

```diff
@@ -0,0 +1,206 @@
+####################################################################
```

Comment:
Might be wroth adding a note that this is a template file, and that the `{{}}` are replaced via sed in run.sh. Maybe we should also rename the file to something like config.template.toml?

Also, definitely slightly concerned rolling our own "template engine" (if you can call a sed 1 liner that), especially if any of these values end up coming from user input.

Don't think we are doing any escaping so I could imagine ways to inject malicious config values.